tests/test_routes.py

```diff
@@ -350,6 +350,7 @@ def test_query_inventories_by_name(self):
         name_list = [record for record in records if record.name == test_name]
         logging.info("Name=%s: %d = %s", test_name, len(name_list), name_list)
         resp = self.client.get(BASE_URL, query_string=f"name={quote_plus(test_name)}")
+        resp
```

Comment:
is this a typo? better to rename resp -> response

Reply:
oh yeah I was testing this! let me remove it